README.md

```diff
@@ -104,6 +104,7 @@ For enhanced file previews (with `scope.sh`):
 * `mediainfo` or `exiftool` for viewing information about media files
 * `odt2txt` for OpenDocument text files (`odt`, `ods`, `odp` and `sxw`)
 * `python` or `jq` for JSON files
+* `sqlite3` and/or `sqlite-utils` for listing tables in SQLite database
```

Comment:
If `sqlite3` is not installed, it will raise command not found error. The sript will not go into the `if` block, just end with `exit 1`.
`sqlite-utils` is optional for fancier box drawing.

Reply:
Then this should be updated to say so rather than "and/*or*" which suggests just `sqlite-utils` is fine.